### PR TITLE
[Woo POS] M2: Update the floating toolbar design and add pop up menu

### LINKED ISSUE #12018: [Woo POS] M2: Update the floating toolbar design. Add pop up menu
Image

### PR DESCRIPTION
**Pull request by kidinov**
Closes: #12018

### Description

The PR adds pop up menu with "exit pos" and "get support" buttons. Get support action itself is not implemented

TfaZ4LUkEwEGrxfnEFzvJj-fi-2877_89425

### Testing information

Using the floating toolbar, open/hide pop-up menu. Try buttons. Pay attention on the UI

### Images/gif

https://github.com/user-attachments/assets/f096cc64-252c-4548-b0cc-a0fcd201420d

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is larger than 300 lines of changes. Please consider splitting it into smaller PRs for easier and faster reviews.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WooCommerce/src/test/kotlin/com/woocommerce/android/ui/woopos/root/WooPosRootViewModelTest.kt [hunk 1]
```diff
@@ -145,5 +135,123 @@ class WooPosRootViewModelTest {
...
+        )
+        assertThat(sut.rootScreenState.value.menu).isEqualTo(WooPosRootScreenState.Menu.Hidden)
+    }
+
+    @Test
+    fun `when OnOutsideOfToolbarMenuClicked, should hide menu if visible`() = runTest {
+        // GIVEN
+        val sut = createSut()
+        sut.onUiEvent(WooPosRootUIEvent.OnToolbarMenuClicked)
+        assertThat(sut.rootScreenState.value.menu).isEqualTo(
```
Comment (AnirudhBhat) [inline WooCommerce/src/test/kotlin/com/woocommerce/android/ui/woopos/root/WooPosRootViewModelTest.kt:205]: **Nitpick:** The approach I'm describing is opinionated, but I believe it addresses a couple of issues with asserting in the GIVEN section:

1. It becomes difficult to pinpoint the cause of a test failure.
2. The test ends up doing more than it claims to be testing.

Instead, consider writing a separate test to assert that the view model state holds the correct value on an event (I see it already being written just above). This way, each test has a single assert, making it clear why a test failed.

As I mentioned, this is just my opinion, and I don't expect you to change your approach if you have a different perspective.

**Review by AnirudhBhat — APPROVED** [review]
Thanks for working on this @kidinov LGTM. I've left one nitpick that is not blocking for merge. :shipit:

File: WooCommerce/src/test/kotlin/com/woocommerce/android/ui/woopos/root/WooPosRootViewModelTest.kt (hunk 1 above)
Comment (kidinov) [reply WooCommerce/src/test/kotlin/com/woocommerce/android/ui/woopos/root/WooPosRootViewModelTest.kt:205]: Those are all good points, and I'll implement them in the following PR. I am moving the toolbar from the Root to Home now

**Pull request merged**